### DISCUSSION TITLE
Display Terminal Output On Touchscreen

### DISCUSSION THREAD

**Discussion by bach37strad** [UnchartedBull/OctoDash#1495] [Ideas]
I've just started using Klipper on my Ender 3 V2, and I'm loving the SCREWS_TILT_CALCULATE function.

I can make a custom button to run the command from the touchscreen easily enough, but i would love to be able to see the output either as a prompt or just in a separate terminal tab in the control panel. 

Is there any way to do this currently in V2 or any plans for something similar in V3?

**Comment by UnchartedBull** [comment]
#757. Octoprint unfortunately does not return the output of a GCode if it is send via the API. So something for after v3.
